pkg/runtime/conversion_generation_test.go

```diff
@@ -143,7 +145,7 @@ func compareBuffers(t *testing.T, generatedFile string, existing, generated byte
 }
 
 func TestNoManualChangesToGenerateConversions(t *testing.T) {
-	versions := []string{"v1beta3"}
+	versions := []string{"v1beta3", "v1"}
```

Comment:
We should change this to testapi.Version when we start running all tests for the v1 api.